v3/src/components/graph/adornments/movable-value/movable-value-adornment-component.tsx

```diff
@@ -140,16 +129,16 @@ export const MovableValueAdornment = observer(function MovableValueAdornment(pro
     const axisMin = isVertical.current ? xScale.domain()[0] : yScale.domain()[0]
     const axisMax = isVertical.current ? xScale.domain()[1] : yScale.domain()[1]
     let newValue = xAttrType === "numeric"
-      ? xScale.invert(event.x) * xCellCount
-      : yScale.invert(event.y) * yCellCount
+      ? xScale.invert(event.x) * cellCounts.x
+      : yScale.invert(event.y) * cellCounts.y
 
     // If the value is dragged outside plot area, reset it to its initial value
     if ((preDragValue != null) && (newValue < axisMin || newValue > axisMax)) {
       newValue = preDragValue
     }
     model.updateDrag(newValue, instanceKey, index)
     refreshValue(newValue, index)
-  }, [getValues, instanceKey, model, refreshValue, xAttrType, xCellCount, xScale, yCellCount, yScale])
+  }, [getValues, instanceKey, model, refreshValue, xAttrType, cellCounts.x, xScale, cellCounts.y, yScale])
```

Comment:
I would think you could use:
```suggestion
  }, [getValues, instanceKey, model, refreshValue, xAttrType, cellCounts, xScale, yScale])
```